runtime/integration-tests/src/test_macros.rs

```diff
@@ -8,39 +8,59 @@ macro_rules! transfer_20_relay_token_from_relay_chain_to_parachain {
 		$parachain_id:ident,
 		$tx_fee:ident
 	) => {{
-		use crate::mock::{units, ALICE};
+		use xcm_emulator::{Network, TestExt};
+		use crate::mock::{units};
 		use frame_support::traits::fungibles::Inspect;
 		use polkadot_core_primitives::Balance;
 		use xcm::latest::{Junction, Junction::Parachain, Junctions::Here};
 		use $para_runtime::CurrencyId;
 
+		use integration_tests_common::constants::accounts;
+		let alice_account_id = accounts::init_balances()[0].clone();
+
 		$mocknet::reset();
-		let transfer_amount: Balance = units(20);
+		let transfer_amount: Balance = units(100);
 		let mut orml_tokens_before = 0;
 
 		// get ALICE's balance before the transfer
 		$parachain::execute_with(|| {
-			orml_tokens_before = $para_runtime::Tokens::balance(CurrencyId::XCM(0), &ALICE.into());
+			orml_tokens_before = $para_runtime::Tokens::balance(CurrencyId::XCM(0), &alice_account_id.clone().into());
+		});
+
+
+		// get ALICE's balance in the relay chain, before the transfer. Ensure it is enough to transfer.
+		$relaychain::execute_with(|| {
+			let before_alice_free_balance = $relay_runtime::Balances::free_balance(&alice_account_id.clone().into());
+			assert!(before_alice_free_balance > transfer_amount);
 		});
 
+
 		// execute the transfer from relay chain
 		$relaychain::execute_with(|| {
+
 			assert_ok!($relay_runtime::XcmPallet::reserve_transfer_assets(
-				$relay_runtime::RuntimeOrigin::signed(ALICE.into()),
-				Box::new(Parachain($parachain_id).into_versioned()),
+				$relay_runtime::RuntimeOrigin::signed(alice_account_id.clone().into()),
+				Box::new(Parachain($parachain_id).into()),
 				Box::new(
-					Junction::AccountId32 { network: None, id: ALICE }
+					Junction::AccountId32 { network: None, id: alice_account_id.clone().into() }
 						.into_location()
 						.into_versioned()
 				),
 				Box::new((Here, transfer_amount).into()),
 				0
 			));
+
 		});
 
 		// a "Deposited" event occurred is proof that the transfer was successful
 		$parachain::execute_with(|| {
 			use $para_runtime::{RuntimeEvent, System};
+
+			for event in System::events() {
+				println!{"Parachain events when receiving of relay token"}
+				println!{"{:?}", event}
+			}
+
```

Comment:
Should we remove these logs again?

Reply:
I left it in case we want to explore the issue regarding the difference in fees I described in the description, of course before merging we can remove.